web/wallassets/manifest.json

```diff
@@ -63,14 +59,9 @@
   "public.css": "public.css",
   "public.css.map": "public.css.map",
   "public.js": "public.js",
-  "themes/_global/img/bg-select.png": "themes/_global/img/bg-select.png",
   "themes/_global/img/icons/carrot-icon--black.png": "themes/_global/img/icons/carrot-icon--black.png",
-  "themes/_global/img/icons/carrot-icon--white.png": "themes/_global/img/icons/carrot-icon--white.png",
-  "themes/_global/img/icons/Diaspora-asterisk.svg": "themes/_global/img/icons/Diaspora-asterisk.svg",
   "themes/_global/img/icons/diaspora-icon--black.png": "themes/_global/img/icons/diaspora-icon--black.png",
   "themes/_global/img/icons/scuttle.png": "themes/_global/img/icons/scuttle.png",
   "themes/_global/img/icons/shaarli.png": "themes/_global/img/icons/shaarli.png",
-  "themes/_global/img/icons/unmark-icon--black.png": "themes/_global/img/icons/unmark-icon--black.png",
```

Comment:
`unmark-icon--black.png` seems also used by the material theme

Reply:
The line is available just below.